dom0/argo.bats

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bats
+
+@test "check sshargo to NDVM" {
+    [ "$(sshargo -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no Network echo hello)" = "hello" ]
+}
+
+@test "check sustained transfer over sshargo to NDVM" {
+    run sshargo -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no Network dd if=/dev/zero bs=1M count=50 >/dev/null
```

Comment:
I think you want to re-work this to get an actually argo transfer rate.  Run dd on the receiving side so you measure rate coming off sshargo.
`run sshargo -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no Network dd if=/dev/zero bs=1M | dd of=/dev/zero bs=1M count=50`
or
`run sshargo -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no Network cat /dev/zero | dd of=/dev/zero bs=1M count=50 iflag=fullblock`
cat + dd needs the iflag or it stops early with a partial read.

I don't hit 50MB/s :(.  There is ssh overhead doing this.

Reply:
Actually, it may be fine how you wrote it.  The listener dd would start counting time before receiving data, so ssh setup & login time overhead made the transfer rate look worse.  Your way only start dd when ssh is already established.

Sorry for the noise.

Reply:
Thanks for the comments - they're not noise. Am looking into this as the ssh invocation appears to need to be piped to another command in order to actually drive the output over the inter-VM connection, which is needed. Will update with a new revision after investigation.

Reply:
Refreshed the PR with a test that checks that the rate of transfer over the sshargo connection exceeds 25 MB/s, a fairly conservative threshold.

Reply:
ok, one more update: now checking the rates reported by both ends of the connection.